android/src/main/java/com/frontegg/reactnative/FronteggRNModule.kt

```diff
@@ -164,7 +164,7 @@ class FronteggRNModule(val reactContext: ReactApplicationContext) :
   }
 
   @ReactMethod
-  fun directLoginAction(type: String, data: String, promise: Promise) {
+  fun directLoginAction(type: String, data: String, ephemeralSession:Boolean, promise: Promise) {
```

Comment:
`ephemeralSession` is it for iOS only?
FronteggRNModule.directLoginAction is never use this param...

Reply:
Current our android not supporting epthemeralSession option.
And cannot build different bridges for android and ios.